_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+        document.getElementById('overlay').style.display = 'none'
+      }
+    </script>
+  </head>
+  <body onload="registerShortcut({id: 'singleShortcut', shortcutKey: '+'}); activateShortcuts();">
+    <div
+      style="
+        display: none;
+        position: fixed;
+        top: 2em;
+        left: 10em;
+        background-color: #505050;
+        color: white;
+        padding: 1em;
+        padding-top: 0em;
+      "
+		  id="overlay"
+	  >
+      <p>Disable/remap shortcut</p>
+      <label>
+        <input type="checkbox" onclick="toggleDisabled('singleShortcut', !this.checked)" checked>
+        Toggle single character keyboard shortcut
+      </label>
+      <br>
+      <label>
+        <input id="remap" type="checkbox" onclick="toggleModifier('singleShortcut', this.checked)">
+        Use "ctrl" key together with the "+" key
+      </label>
+      <br>
+      <button onclick="closeModal();">Dismiss</button>
+    </div>
+    
+    <input type="button" onclick="openModal()" value="Open modal" />
+    
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+This document is not an [HTML document][].
+
+```html
+<svg height="100" width="100">
+  <circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />
+  Sorry, your browser does not support inline SVG.  
+</svg>
+```
+
+#### Inapplicable Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `true`.
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 2</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({shortcutKey: '+', ctrlKey: true}); activateShortcuts();">
+    <label for="target">Add to list (press "ctrl" and "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [non-printable character][non-printable characters].
+
+```html
+<html>
+  <head>
+    <title>Inapplicable Example 3</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({ctrlKey: true}); activateShortcuts();">
+    <label for="target">Add to list (press "ctrl" and "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Inapplicable Example 4
+
+This [HTML document][] has a [keyboard event][] that is not [dispatched][] to an [event target][].
```

Comment:
I know I've been pushing for that example back in the days, but I realise it is incorrect (event are always dispatched to an event target) and un-needed (we've removed that part of the applicability).